script.js

```diff
@@ -21,4 +21,187 @@ function closeScreen() {
+                </div>
+            </div>
+            
+            <div class="modal-bottom">
+                <p>${popData[data].Description}</p>
+            </div>
+            <div class="buttons-1">
+                <button class="modal-11">See live <img src="${popData[data].liveIcon}" alt="live logo icon"></button>
+                <button class="modal-12">See Source <img src="${popData[data].githubIcon}" alt="github logo"></button>
+            </div>
+        </div>
+    </div>
+  `;
+  // Adding event listeners to respective buttons
+
+  const modalCloseButton = document.querySelector('.modal-top');
+  function toggleModal() {
+    if (getComputedStyle(modal).display === 'flex') {
+      modal.classList.add('modal-hide');
+      setTimeout(() => {
+        modal.style.display = 'none';
+        modal.classList.remove('modal-show', 'modal-hide');
+      }, 200);
+    } else {
+      modal.style.display = 'flex';
+      modal.classList.add('modal-show');
+    }
+  }
+
+  modalCloseButton.addEventListener('click', () => toggleModal());
+
+  modal.addEventListener('click', (event) => {
+    if (event.currentTarget === event.target) toggleModal();
+  });
+}
+
+// Adding work cards dynamically
+const workProfile = document.querySelector('.work-profiles-10');
+
+for (let i = 0; i < 3; i += 1) {
+  workProfile.innerHTML += `
+  <div class="work-profile-1 ${popData[i].classNO}">
+  <div class="works-text">
+      <div class="works-title">
+          <h3>
+          Multi-Post Stories<br>
+          Gain+Glory
+          </h3>
+      </div>
+      <div class="works-tags">
+          <ul class="tags">
+              <li class="tech-tag">Ruby on Rails</li>
+              <li class="tech-tag">CSS</li>
+              <li class="tech-tag">Javascript</li>
+              <li class="tech-tag">HTML</li>
+          </ul>
+      </div>
+      <div class="works-cta">
+          <button type="submit" class="project-button" onclick="modalSection(${i})">
+              See Project
+          </button>
+      </div>
+  </div>
+</div>
+  `;
+}
+
+const workProfile2 = document.querySelector('.work-profiles-20');
+
+for (let i = 3; i < popData.length; i += 1) {
+  workProfile2.innerHTML += `
+  <div class="work-profile-1 ${popData[i].classNO}">
+  <div class="works-text">
+      <div class="works-title">
+          <h3>
+          Multi-Post Stories<br>
+          Gain+Glory
+          </h3>
+      </div>
+      <div class="works-tags">
+          <ul class="tags">
+              <li class="tech-tag">Ruby on Rails</li>
+              <li class="tech-tag">CSS</li>
+              <li class="tech-tag">Javascript</li>
+              <li class="tech-tag">HTML</li>
+          </ul>
+      </div>
+      <div class="works-cta">
+          <button type="submit" class="project-button" onclick="modalSection(${i})">
+              See Project
+          </button>
+      </div>
+  </div>
+</div>
+  `;
+}
```

Comment:
- [ ] Kindly follow  design for  mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). Please try to adjust the alignment based on the Figma design. Minimum mobile screen size should be 375px.
The Figma design Look likes: 
![cap1](https://user-images.githubusercontent.com/93598189/190662695-a9518eab-38fc-4d6a-8724-a47b2376d8a7.PNG)
![Capture](https://user-images.githubusercontent.com/93598189/190662711-e240792d-3970-476d-afec-19b854b61391.PNG)

Your design at Minimum mobile screen size ( 375px) Look likes this : 
![Cap2](https://user-images.githubusercontent.com/93598189/190663748-ca37f288-956f-496d-8026-a818df481a1b.PNG)

The Pop up is not fully responsive at the bottom "See Live " and "See Source code " is not visible. Kindly adjust it.

![Captur1](https://user-images.githubusercontent.com/93598189/190664087-87df90a5-ecff-4987-b22e-d48e86587873.PNG)